pytest_pylint.py

```diff
@@ -111,6 +111,11 @@ class PyLintItem(pytest.Item, pytest.File):
     def __init__(self, fspath, parent, msg_format=None, pylintrc_file=None):
         super(PyLintItem, self).__init__(fspath, parent)
 
+        if hasattr(self, 'add_marker'):
```

Comment:
I just copied the code from pytest-flakes. Meanwhile I checked in the pytest repo that the add_marker function was added in version 2.4.2. Therefor I will simplify the patch.